fuzzing.cpp

```diff
@@ -127,9 +127,11 @@ int main(int argc, char *argv[])
     char chr;
     std::ifstream file;
     std::stringstream buffer;
-
+#ifndef COMPILE_WITH_GCC
+    //Do not use __AFL_LOOP when using gcc
```

Comment:
I generally prefer to have comments next to the particular line of code instead of in front if it. I will reformat this.